Makefile-docker

```diff
@@ -156,13 +158,13 @@ setup-ui-tests:
 	# Create a proper superuser that can be used to access the API
 	python manage.py waffle_switch super-create-accounts on --create
 	python manage.py waffle_switch activate-autograph-signing on --create
-	python manage.py createsuperuser --email=uitest@restmail.net --username=uitest --noinput --add-to-supercreate-group --hostname=olympia-frontend.test
 
 ui-tests: setup-ui-tests
 	# Generate test add-ons and force a reindex to make sure things are updated
+	pip install --progress-bar=off --no-deps -r requirements/uitests.txt
 	python manage.py generate_ui_test_addons
 	python manage.py reindex --force --noinput --wipe
-	pip install --progress-bar=off --no-deps -r requirements/uitests.txt
+	pytest --driver Firefox -n 2 tests/ui/
```

Comment:
Parallel tests! Did you test if we get an actual performance benefit here?

Reply:
Yeah its a good bit faster. Didn't get any hard numbers though.